scene/animation/animation_tree.cpp

```diff
@@ -995,7 +996,7 @@ void AnimationTree::_process_graph(double p_delta) {
 					case Animation::TYPE_POSITION_3D: {
 #ifndef _3D_DISABLED
 						TrackCacheTransform *t = static_cast<TrackCacheTransform *>(track);
-						if (track->root_motion) {
+						if (track->root_motion && calc_root) {
```

Comment:
AnimationTree performs a seek to change the playback time, but should be able to choose whether to calculate the root motion or not.